app/views/accounts/show.html.haml

```diff
@@ -3,16 +3,14 @@
 -#  See the COPYING file at the top-level directory or at
 -#  https://github.com/puzzle/cryptopus.
 
-= link_to 'Edit', edit_account_path(@account.id), class: 'btn btn-primary pull-right'
+= link_to image_pack_tag("edit.svg"), edit_account_path(@account.id), id: 'edit_account_button', class: 'btn btn-secondary pull-right'
 = link_to 'Move', '#', class: 'btn btn-primary pull-right', id: 'move_account_button'
 %h1= t '.title', account_name: @account.accountname
 %div
+  %p.description-text
+    = t '.last_change', time_ago: (time_ago_in_words @account.updated_at)
   %p
-    %strong= t :description
-    = simple_format @account.description
-  %p
-    %strong= t :tag
-    = simple_format @account.tag
+    = h @account.description
```

Comment:
nice, we're escaping html  here. https://api.rubyonrails.org/classes/ERB/Util.html#method-c-h
do we also sanitize this content when adding/editing? should we do this in the future ?